test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -2071,55 +2073,65 @@ def create_temp_dir(self, hostname=None, suffix='', prefix='PtlPbs',
         :type suffix: str
         :param prefix: the directory name will begin with this prefix
         :type prefix: str
-        :param dir: the directory will be created in this directory
-        :type dir: str or None
-        :param uid: Optional username or uid of temp directory owner
-        :param gid: Optional group name or gid of temp directory
+        :param dirname: the directory will be created in this directory
+        :type dirname: str or None
+        :param asuser: Optional username of temp directory owner
+        :type asuser: str
+        :param asgroup: Optional group name of temp directory
                     group owner
+        :type asgroup: str
         :param mode: Optional mode bits to assign to the temporary
                      directory
+        :type mode: octal integer
         :param level: logging level, defaults to INFOCLI2
         """
+        current_user_info = self.get_id_info(self.get_current_user())
```

Comment:
Optional (But I would highly recommend doing): You can make use of PbsUser.get_user

Reply:
I can do so but I think it will have a limitation that the user who is running PTL will then have to be one of the users PTL creates. Currently, I can run tests as any system user and it works.

Reply:
I don't think PTL fw/utils or any tests go beyond what we have in `PBS_ALL_USERS`, and if we really need to go beyond `PBS_ALL_USERS` then we should enhance `get_user` to allow it.

Reply:
Well, like in this case, If I run a test as "arung" it will fail because current_user (arung) is not in the list of PBS_ALL_USERS. Is there anything that says a non-ptl user cannot run a PTL test? 
Making this change will force to run the test as one of the PBS_ALL_USERS.

Reply:
Sorry missed you comment somehow: No we don't have any such guidelines which say you can't run PTL as use which is not part of PBS_ALL_USERS. But usually people write tests using users which is listed on PBS_ALL_USERS so there are high chances of failures if you run PTL as a user which not listed in PBS_ALL_USERS.